qldbdriver/statement_execution_test.go

```diff
@@ -448,6 +448,78 @@ func TestStatementExecution(t *testing.T) {
 		assert.True(t, ok)
 	})
 
+	t.Run("Execution metrics for stream result", func(t *testing.T) {
+		driver, err := testBase.getDriver(ledger, 10, 4)
+		require.NoError(t, err)
+		defer driver.Shutdown(context.Background())
+		defer cleanup(driver, testTableName)
+
+		// Insert docs
+		_, err = driver.Execute(context.Background(), func(txn Transaction) (interface{}, error) {
+			return txn.Execute(fmt.Sprintf("INSERT INTO %s << {'col': 1}, {'col': 2}, {'col': 3} >>", testTableName))
+		})
+		require.NoError(t, err)
+
+		selectQuery := fmt.Sprintf("SELECT * FROM %s as a, %s as b, %s as c, %s as d, %s as e, %s as f",
```

Comment:
Why aren't reusing params ala `?`?

Reply:
This statement is not using parameters.

If I were to update to:
```
selectQuery := "SELECT * FROM ? as a, ? as b, ? as c, ? as d, ? as e, ? as f"

result, err := txn.Execute(selectQuery, testTableName, testTableName, testTableName, testTableName, testTableName, testTableName)
```

It returns:
` BadRequestException: Unsupported PartiQL Statement`

Reply:
Ok, I misread this